NickvisionTubeConverter.GNOME/Program.cs

```diff
@@ -35,9 +35,11 @@ public Program(string[] args)
         _mainWindow = null;
         _mainWindowController = new MainWindowController();
         _mainWindowController.AppInfo.Changelog =
-            @"* Added an option to disable subtitle embedding
-              * Fixed an issue where an embedded subtitle was empty
-              * If embedding a subtitle fails, it will now be saved as a separate file
+            @"* Added an option to disable embedding subtitles in a file and instead download them in a separate file
+              * Added the ability to specify ""all"" in the subtitle languages list to download all available languages
+              * Fixed an issue where empty subtitles were sometimes embedded
+              * Fixed an issue where arte.tv links were not validating
```

Comment:
That was yt-dlp bug, maybe just say that yt-dlp was updated in Flatpak and Snap fixing some issues?

Reply:
Since this issue was opened on our app, I mentioned it specifically. 

Maybe `Fixed an issue where arte.tv links were not validating thanks to a yt-dlp update`?

Reply:
okay 👍🏻